client/src/Pages/Dashboard/DashboardPage.js

```diff
@@ -17,7 +17,7 @@ export function DashboardPage(props) {
 
   return (
     <ApolloProvider client={client}>
-      <DashboardView {...props} />
+        <DashboardView  />
```

Comment:
it seems like removing the `{...props}` may break features. Let me know the reason for this change.

Reply:
I've removed props it doesn't break anything for me.
It might of been when handle change and selected option were deleted because i'm passing it as a callback function to car List, car list then uses the method i called actions as props.action but i could be misunderstanding stuff.